IgOmeProfiling_pipeline.py

```diff
@@ -170,9 +166,10 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
     parser.add_argument('--tfidf_method', choices=['boolean', 'terms', 'log', 'augmented'], default='boolean', help='TF-IDF method')
     parser.add_argument('--tfidf_factor', type=float, default=0.5, help='TF-IDF augmented method factor (0-1)')
     parser.add_argument('--shuffles', default=5, type=int, help='Number of controlled shuffles permutations')
-
+    parser.add_argument('--stop_machines', action=True, help='Do turn off the machines in AWS at the end of the running?')
+    parser.add_argument('--type_machines_to_stop', defualt='all', type=str, help='To stop machines by type. string with _ for separate.')
```

Comment:
`--machine_types_to_stop`
Default should be empty string, not `all`.
Should be: Type of machines to stop, separated by comma. Empty value means all machines. Example: ...